test/Staking.ts

```diff
@@ -133,15 +157,15 @@ describe("Staking contract", function () {
       ).to.be.revertedWith("Only staker can call function");
     });
 
-    it("should failed if current number of validators equals to MINIMUM_REQUIRED_NUM_VALIDATORS", async () => {
+    it("should failed if current number of validators equals to MinimumRequiredNumValidators", async () => {
```

Comment:
This test case name should be more user friendly, not reference literal variable names

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9